presentation/src/main/java/com/anytypeio/anytype/presentation/onboarding/signup/OnboardingVoidViewModel.kt

```diff
@@ -80,16 +80,14 @@ class OnboardingVoidViewModel @Inject constructor(
 
     private fun proceedWithCreatingAccount() {
```

Comment:
please add `Deprecated("To be deleted")` to this view model constructor

Reply:
done